internal/pkg/externalplugins/lgtm/lgtm.go

```diff
@@ -50,16 +51,15 @@ func HelpProvider(epa *externalplugins.ConfigAgent) func(
 			}
 		}
 		pluginHelp := &pluginhelp.PluginHelp{
-			Description: "The ti-community-lgtm plugin manages the application and " +
-				"removal of the 'status/LGT{number}' (Looks Good To Me) label which is typically used to gate merging.",
-			Config: configInfo,
+			Description: "The ti-community-lgtm plugin manages the 'status/LGT{number}' (Looks Good To Me) label.",
+			Config:      configInfo,
 		}
 
 		pluginHelp.AddCommand(pluginhelp.Command{
-			Usage:       "/lgtm [cancel] or GitHub Review action",
```

Comment:
Please revert the `or GitHub Review action`. Because it also work.

Reply:
done